op-challenger/challenger/output_test.go

```diff
@@ -10,13 +10,49 @@ import (
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/log"
 
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
 	"github.com/ethereum-optimism/optimism/op-challenger/metrics"
 	"github.com/ethereum-optimism/optimism/op-node/eth"
 	"github.com/ethereum-optimism/optimism/op-node/testlog"
 )
 
+func TestParseOutputLog_Succeeds(t *testing.T) {
```

Comment:
Can we test against a real log?

Reply:
I'm not really sure how best to do this. Is there a way to encode a sol `emit OutputProposed(...)` using the abi encoder?

Reply:
hmm, it doesn't look like there is an easy way to do this. We could run the EVM in memory. Another option is to hook this up to an e2e test. I think this log looks correct, but I'm not gonna be 100% sure until I see it running in the full system

Reply:
I think this is sufficiently tested to continue work ontop in trunk for now